config/config.exs

```diff
@@ -53,3 +53,13 @@ config :guardian, Guardian.DB,
   token_types: ["access"],
   # default: 60 minute
   sweep_interval: 60
+
+# Configure :ex_aws
+config :ex_aws,
+  access_key_id: {:system, "AWS_ACCESS_KEY_ID"},
+  secret_access_key: {:system, "AWS_SECRET_ACCESS_KEY"},
+  s3: [
+    scheme: "https://",
+    host: "sreyansapitest.s3-ap-southeast-1.amazonaws.com",
```

Comment:
Please add a corresponding example in `prod.secrets.exs` as a template for production.